### PR TITLE
fix(plugin-meetings): fix direct info uri

### PR DESCRIPTION
**Pull request by Coread**
# COMPLETES #< INSERT LINK TO ISSUE >

https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-364846

## This pull request addresses

This fix uses the `uri` paramter when creating the request params rather than the non-existent `directURI` parameter

## by making the following changes

< DESCRIBE YOUR CHANGES >

### Change Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [ ] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.js [hunk 1]
```diff
@@ -192,14 +192,18 @@ export default class MeetingInfoV2 {
...
 
     const directURI = await MeetingInfoUtil.getDirectMeetingInfoURI(destinationType);
 
-    if (directURI) options.directURI = directURI;
+    if (directURI) {
+      options.uri = directURI;
+    }
+    else {
+      options.service = 'webex-appapi-service';
+      options.resource = 'meetingInfo';
```
Comment (maxinteger) [inline packages/node_modules/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.js:205]: is this the same value as `MEETINGINFO` in `packages/node_modules/@webex/plugin-meetings/src/constants.ts`? If yes then please use the constant.

File: packages/node_modules/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.js [hunk 2]
```diff
@@ -192,14 +192,18 @@ export default class MeetingInfoV2 {
...
     };
 
     const directURI = await MeetingInfoUtil.getDirectMeetingInfoURI(destinationType);
 
-    if (directURI) options.directURI = directURI;
+    if (directURI) {
+      options.uri = directURI;
+    }
+    else {
+      options.service = 'webex-appapi-service';
```
Comment (maxinteger) [inline packages/node_modules/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.js:204]: Looks like `'webex-appapi-service'` used in many places we should have constant for this probably in `packages/node_modules/@webex/plugin-meetings/src/constants.ts`

**Review by maxinteger — CHANGES_REQUESTED** [review]

File: packages/node_modules/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.js (hunk 1 above)
Comment (Coread) [reply packages/node_modules/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.js:205]: this represents part of a url. A resource could be meetingInfo/something/something else. I don't think we want to use constants for this. It is a coincidence that it is the same as the one used for the locus meeting info

File: packages/node_modules/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.js (hunk 2 above)
Comment (Coread) [reply packages/node_modules/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.js]: I've changed this to be a constant

**Review by robstax — APPROVED** [review]

**Pull request merged**